src/test/resources/application.properties

```
@@ -5,3 +5,6 @@ spring.datasource.password=password
 spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
 
 spring.h2.console.enabled=true
+
+jwt.expiration=900000
+jwt.secret=hellomates1234567890asdfghjklwertyuisdsdsdsdsd
```

```suggestion
jwt.secret=hellomates1234567890asdfghjklwertyuisdsdsdsdsd

```
always have one empty line at the EOF